src/scss/utils/mixins.scss

```diff
@@ -31,3 +31,121 @@
     text-shadow: none;
   }
 }
+
+/**
+ * This is where all your colours are declared.
+ *
+ * The key should be equivalent to the name of the
+ * class to apply the theme colour, which is in the
+ * format of 'theme-{$element}-{$color}'.
+ *
+ * Classes should be declared on the body so that they
+ * can be accessed and applied to any element.
+ */
+$theme-colors: (
+  red: #E51016,
+  dark-red: #AD161E,
+  pink: #B32A95,
+  purple: #6239C8,
+  blue: #1F6BFE,
+  dark-blue: #123581,
+  default-accent: $brand-primary,
+  default-background: $theme-bg,
+  teal: #00837A,
+  green: #298436,
+  dark-orange: #D34300,
+  dark-ochre: #947200,
+  black: #111,
+  dark-grey: $gray,
+  light-grey: #EAEAEA,
+  white: #FFF,
+);
+
+/**
+ * Theme header color
+ */
+@mixin theme-header-color {
+  @each $name, $color in $theme-colors {
+    .theme-header-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme nav color
+ */
+@mixin theme-nav-color {
+  @each $name, $color in $theme-colors {
+    .theme-nav-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme footer color
+ */
+@mixin theme-footer-color {
+  @each $name, $color in $theme-colors {
+    .theme-footer-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme carousel color.
+ * The background colour of the carousel when there is no image set.
+ */
+@mixin theme-carousel-color {
+  @each $name, $color in $theme-colors {
+    .theme-carousel-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme accent color which defaults to background color.
+ * Affects colour of buttons, current navigation items etc.
+ *
+ * You can specify a different CSS property to use when
+ * calling the mixin e.g. @include theme-accent-color('color').
+ *
+ * You can also specify the hover/focus color of an element.
+ * You can choose between lighten and darken, but this could
+ * be extended to add other color manipulation techniques.
+ *
+ * This defaults to none so as not to bloat the CSS when there
+ * is no hover state for an element.
+ */
+@mixin theme-accent-color($property: 'background-color', $type: 'none', $amount: 0%) {
+  @each $name, $color in $theme-colors {
+    body.theme-accent-#{$name} & {
+      #{$property}: $color;
+
+      @if ($type != 'none') {
+        &:hover,
+        &:focus {
```

Comment:
Rather than including `:hover, :focus` inside this mixin, it may be better to not assume the type is only applied to hover/focus and instead specify that in each use-case. e.g.

```
@mixin theme-accent-color($property: 'background-color', $modify: 'none', $amount: 0%) {
  @each $name, $color in $theme-colors {
    body.theme-accent-#{$name} & {
      @if ($modify == 'none') {
        #{$property}: $color;
      } @else if ($modify == 'lighten') {
        #{$property}: lighten($color, $amount);
      } @else if ($modify == 'darken') {
        #{$property}: darken($color, $amount);
      }
    }
  }
}

...

a {
  @include theme-accent-color('border-bottom-color');

  &:hover {
    @include theme-accent-color('border-bottom-color', 'darken', 10%);
  }
}
```